charts/datagovuk/templates/publish/init-configmap.yaml

```diff
@@ -8,9 +8,12 @@ data:
       sleep 1;
     done
 
+    ## no permissions to create database so remove
+    # bin/rails db:setup
+    # bundle exec rails db:create && 
+
```

Comment:
Please remove commented-out lines, or if they're there to serve as a reminder for something that you're about to do then please make them a TODO.

Reply:
👍